src/library/templates/library/library_quests.html

```diff
@@ -1 +1 @@
-{% extends "quest_manager/quests.html" %}
+{% include "quest_manager/quests.html" with view_type=-1%}
```

Comment:
Why is this switched to `include` instead of `extend`? That seems significant

Reply:
you cant use the "with" template tag with `extends`. If `view_type` and `VIEW_TYPES` are empty it causes all tabs to be active since `None == None`.

An alternative approach could be adding `view_type` to library view context

Reply:
include is a completely different construct than extends.  And I'm sure you can use "with" inside an extends, every template in the app extends from base_sidebar.html with extends base.html, which extends from.  Something else is wrong with your code that is causing you problems. For example check out this one (but there are many extends + with in templates):
badges/templates/badges/detail.html